module_utils/azure_rm_common.py

```diff
@@ -322,20 +339,34 @@ def __init__(self, derived_arg_spec, bypass_checks=False, no_log=False,
                                                                      cloud_environment=self._cloud_environment,
                                                                      verify=self._cert_validation_mode == 'validate')
 
-        elif self.credentials.get('ad_user') is not None and self.credentials.get('password') is not None:
-            tenant = self.credentials.get('tenant')
-            if not tenant:
-                tenant = 'common'  # SDK default
-
-            self.azure_credentials = UserPassCredentials(self.credentials['ad_user'],
-                                                         self.credentials['password'],
-                                                         tenant=tenant,
-                                                         cloud_environment=self._cloud_environment,
-                                                         verify=self._cert_validation_mode == 'validate')
+        elif self.credentials.get('ad_user') is not None and \
+                self.credentials.get('password') is not None:
```

Comment:
seems the indent is strange.

Reply:
just follow the first `if` block. is there any common practice for indent of continuatation line in all azure modules?